packages/lesmis/src/theme/contentPageTheme.js

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Would be nice if there were a way to "merge modifications" from one theme into a base theme (this has some overlap with the main LESMIS theme)